packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.tsx

```diff
@@ -17,6 +17,8 @@ export const TagItem = (props: IPickerItemProps<ITag>) => (
     key={ props.index }
     data-selection-index={ props.index }
     data-is-focusable={ !props.disabled && true }
+    onClick={ props.onClick }
+    onKeyDown={ props.onKeyDown }
```

Comment:
okay, if we are modeling Tag as a button that can be activated, we need to add proper role="button" to convey the action. Moreover, at the consumption side you may be interested in passing a single onClick prop rather than handling both mouse and keyboard actions separately. In that case, you need to channel these events to a common method here which calls into prop.onClick()